src/Microsoft.SqlTools.Hosting/Utility/Logger.cs

```diff
@@ -73,6 +73,16 @@ public static class Logger
 
             Logger.isInitialized = true;
 
+            // Create the log directory
+            string logDir = Path.GetDirectoryName(logFilePath);
+            if (!string.IsNullOrWhiteSpace(logDir))
+            {
+                if (!Directory.Exists(logDir))
+                {
+                    Directory.CreateDirectory(logDir);
```

Comment:
should this be in a try/catch block?  or are exceptions already handled\recovered higher up?